src/Randomizer.SMZ3.Tracking/tracker.json

```diff
@@ -1491,6 +1511,7 @@
       "What did you say?",
       "What was that?",
       "Please repeat",
+      "Try again",
       [ "English motherfucker, do you speak it?", 0.05 ]
```

Comment:
Maybe add "Toggle boots on" here too with a very low chance? With or without a "just kidding" at the end.